bin/hermes

```diff
@@ -65,7 +65,7 @@ def request_get(path):
         if response.status_code < 500:
             break
 
-    if response.status_code == requests.codes.not_found:
+    if 400 <= response.status_code  < 500:
         raise HermesNotFound("{} returned 404".format(
```

Comment:
Did you want to fix this exception or at least the message to be more accurate given then encompasses all `Bad Request`s